app/assets/javascripts/admin/modules/unpublish-display-conditions.js

```diff
@@ -17,6 +17,21 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
   UnpublishDisplayConditions.prototype.init = function () {
     this.initUnpublishTypeListener()
+    this.initUnpublishRedirectListener()
+  }
+
+  UnpublishDisplayConditions.prototype.initUnpublishRedirectListener = function () {
+    var checkbox = this.module.querySelector('.govuk-checkboxes__item input[name="unpublishing[redirect]"]')
+
+    checkbox.addEventListener('change', function (e) {
+      var display = e.currentTarget.checked ? 'none' : 'block'
```

Comment:
Minor, no need to change this but for future reference you can use an empty string instead of block. That can be better as it'll default to the elements standard display type whereas 'block' can change the rendering semantics of elements intended to be `display: inline` or `display: inline-block`

Reply:
Thanks, good to know for future tickets!